presto-geospatial/src/test/java/com/facebook/presto/geospatial/TestGeoQueries.java

```diff
@@ -475,4 +476,13 @@ public void testSTWithin()
         assertFunction("ST_Within(ST_GeometryFromText('LINESTRING (1 1, 3 3)'), ST_GeometryFromText('POLYGON ((0 0, 0 4, 4 4, 4 0))'))", BOOLEAN, true);
         assertFunction("ST_Touches(ST_GeometryFromText('MULTIPOLYGON (((1 1, 1 3, 3 3, 3 1)), ((0 0, 0 2, 2 2, 2 0)))'), ST_GeometryFromText('POLYGON ((0 1, 3 1, 3 3, 0 3))'))", BOOLEAN, false);
     }
+
+    @Test
+    public void testInvalidWKT()
+            throws Exception
+    {
+        assertInvalidFunction("ST_LineFromText('LINESTRING (0 0, 1)')", StandardErrorCode.INVALID_FUNCTION_ARGUMENT, "invalid WKT form: LINESTRING (0 0, 1)");
```

Comment:
With the change above the test should be updated too. Please also static import `INVALID_FUNCTION_ARGUMENT`.